src/pages/ItemDetail.jsx

```diff
@@ -0,0 +1,32 @@
+import { useNavigate } from "react-router-dom";
+import CommentList from "../components/CommentList";
+import InquiryInput from "../components/InquiryInput";
+import ProductInfo from "../components/ProductInfo";
+import ic_back from "../assets/ic_back.png";
+import "./ItemDetail.css";
+
+const ItemDetail = () => {
+  const navigate = useNavigate();
+
+  const handleGoBack = () => {
+    if (window.history.length > 1) {
+      navigate(-1);
+    } else {
+      navigate("/items");
+    }
+  };
+
+  return (
+    <div className="item-detail__container">
+      <ProductInfo />
+      <InquiryInput />
+      <CommentList />
+      <button className="item-detail__back-button">
+        <span>목록으로 돌아가기</span>
+        <img src={ic_back} alt="뒤로가기" className="item-detail__back-icon" onClick={handleGoBack} />
+      </button>
```

Comment:
❗️ 수정요청
상호작용이 가능한 button 태그에 onClick 함수를 연결하시는 게 더 적절해보입니다!

```suggestion
      <button className="item-detail__back-button" onClick={handleGoBack}>
        <span>목록으로 돌아가기</span>
        <img src={ic_back} alt="뒤로가기" className="item-detail__back-icon"  />
      </button>
```